app/Import/ImportException.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Import;
```

Comment:
I'd move this exception to the exception namespace. We can keep the "import" subfolder, so we don't have one bloated folder.